packages/page-accounts/src/Accounts/Summary.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017-2021 @polkadot/app-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { AccountBalance } from '../types';
+
+import React from 'react';
+
+import { useTranslation } from '@polkadot/app-treasury/translate';
+import { CardSummary, SummaryBox } from '@polkadot/react-components';
+import { FormatBalance } from '@polkadot/react-query';
+
+interface Props {
+  className?: string;
+  balance?: AccountBalance;
+}
+
+function Summary ({ balance, className }: Props) {
+  const { t } = useTranslation();
+
+  return (
+    <SummaryBox className={className}>
+      {balance?.total.gtn(0) &&
+        <CardSummary label={t<string>('total balance')}>
+          <FormatBalance value={balance?.total} />
```

Comment:
```suggestion
          <FormatBalance value={balance.total} />
```

Here (and below), we have already determined balance exists.

Actually the `balance?.` is quite heavy (read: horribly Babel-wise), so it is even better to do something like -

```js
<SummaryBox ...>
  {balance && (
    <>
      {balance.total.gt(BN_ZERO) && (
        <CardSummary ... />
      )}
    </>
  )}
</SummaryBox>
 ``

Reply:
Done (implemented second option)